xc/common/utils/prjxray_create_place_constraints.py

```diff
@@ -991,7 +991,7 @@ def main():
 
     idelayctrl_instances = place_constraints.get_used_instances("IDELAYCTRL")
 
-    assert len(idelayctrl_cmts) == len(
+    assert len(idelayctrl_cmts) <= len(
```

Comment:
This doesn't feel right? Why is less than now allowed?

Reply:
Yeah, I think this was meant for some situations in which the number of IDELAYCTRLs in the design is less than the number of IDELAY cmts. Basically Vivado can replicate the IDELAYCTRLs if they are needed in different banks, even though there is only one of them explicitly instantiated in the design.